src/references/media_library.md

```diff
@@ -0,0 +1,88 @@
+
+!!! tip "The maximum file size is 5 MB at any resolution size."
+
+![Uploading](../img/MediaLibrary/uploading.png)
+![Uploaded Image](../img/MediaLibrary/uploaded_image.png)
+_File Dialog Browser (left) - Uploaded Image (right)_
+{: .image-cluster}
+
+### Creating Image Assets
+
+Creating an image asset can be done from the Media Library by clicking on an existing image and then clicking the **Create Image Asset**. Creators can also right-click on the image and select **Create Asset**.
+
+Creating an asset will create an image brush in **My Brushes** that will be added to the existing project that can be used.
+
+![!Created Asset](../img/MediaLibrary/created_asset.png){: .center loading="lazy" }
+
+### Renaming Content
+
+Images in the Media Library can be renamed by right-clicking on the image and selecting **Rename**, or from the preview panel in the Media Library. Renaming an image allows creators to keep their Media Library content organized.
+
+Notice how the content in the image below is named according to what they are which makes finding these images much easier as the library gets bigger.
+
+![!Rename Image](../img/MediaLibrary/rename.png){: .center loading="lazy" }
+
+### Deleting Content
+
+Deleting images from the Media Library can be done by right-clicking on the image and selecting **Delete**, or from the preview panel in the Media Library. Deleting an image will delete it from the Media Library, and brushes that have been created using that image in any of the creators projects will show up as white in the game as the image will no longer be valid.
+
+![!Delete Image](../img/MediaLibrary/delete.png){: .center loading="lazy" }
+
+### My Brushes
+
+All assets created from the Media Library can be found in **My Brushes** in the **Project Content** window. These brushes can be used in the UI by dragging them in to the preview window or Hierarchy.
+
+If a brush is not added to an existing UI Container, then one will be created automatically.
+
+![!Add Brush](../img/MediaLibrary/add_brush.png){: .center loading="lazy" }
+
+#### Retaining Aspect Ratio
+
+Creators may notice that brushes which get added to the **Hierarchy** do not maintain the original aspect ratio of the image. By enabling the option **Retain Aspect Ratio** in the Properties window of the UI Image, the aspect ratio will be retained regardless of its given dimensions.
+
+In the image below, the left UI Image does not have **Retain Aspect Ratio** enabled, whereas the right UI image does. This can still be scaled, but will retain the aspect ratio.
+
+![!Ratio](../img/MediaLibrary/ratio.png){: .center loading="lazy" }
+
+#### Brush Picker
+
+The **Brush Picker** window will allow creators to find Core Base Brushes and their own brushes quickly when needing to swap out an existing brush. The Core Base Brushes can be filtered by selecting a type from the filter drop down.
+
+![!Brush Picker](../img/MediaLibrary/brush_picker.png){: .center loading="lazy" }
+
```

Comment:
Did we want to mention Community Content not supporting media at the moment?

Reply:
Yeah, going to add that in. Haven't decided where to put it yet.